supabase/migrations/082_posts_hotpath_indexes.sql

```
@@ -0,0 +1,34 @@
+--   idx_posts_created_at  … posts(created_at DESC)
+--   idx_posts_user_id     … posts(user_id)
+--   idx_posts_parent_id   … posts(parent_id)
+--
+-- 適用後は本番相当データで EXPLAIN ANALYZE を取り、
+-- 実際のプランで Index Scan に切り替わることを確認すること。
+-- ============================================================
+
+-- プロフィール投稿 / フォロー中タイムライン:
+--   eq(user_id) AND parent_id IS NULL ORDER BY created_at DESC LIMIT 50
+CREATE INDEX IF NOT EXISTS posts_user_created_idx
+    ON public.posts (user_id, created_at DESC);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)